scripts/README.md

```diff
@@ -97,42 +97,54 @@ npx truffle exec scripts/transfer_approve_deposit.js --masterSafe=$MASTER_SAFE -
-]
-```
-
-If you have forgotten the addresses of your brackets, then you should read in the next section how to retrieve them.
-
-The script can automatically determine the amount, instead of having to specify it on the file.
-This is achieved by adding the flag `--allTokens` to the withraw command. This is possible in any of the following commands.
+To this end, you must specify the brackets you want to withdraw from with `--brackets` and the tokens to be withdrawn using either `--tokens` (which takes a list of addresses) or `--tokenIds` (which takes a list of token IDs).
 
-Withdrawing is a two-step process: first, withdrawals must be requested on the exchange; then the withdrawals can be executed, and at the same time the funds can be sent back to the master Safe.
+First, a withdraw request must be created for each bracket and token.
+The following command request withdrawing of all DAI and WETH (token ID 7 and 1 respectively) for the brackets at addresses `0x1` and `0x2`:
 
 ```js
-npx truffle exec scripts/withdraw.js --requestWithdraw --masterSafe=$MASTER_SAFE --withdrawalFile="./examples/exampleDepositList.json" --network=$NETWORK_NAME
+npx truffle exec scripts/request_withdraw.js --masterSafe=$MASTER_SAFE --brackets=0x1,0x2 --tokenIds=1,7 --network=$NETWORK_NAME
 ```
 
+This command can be used to halt all tradings involving these brackets and tokens: Even if orders are still up, no trading will be possible starting from the next batch after the corresponding transaction has been confirmed.
+No funds will have moved yet, but a withdraw request will have been registered on the exchange.
+
+The next step transfers the funds from the exchange to the master Safe.
+Internally, it composes two steps together: withdrawing funds from the exchange to each brackets, and then transferring funds from the brackets to master.
+The parameters of the call are the same as before, the only change is the name of the script to be executed:
+
 ```js
-npx truffle exec scripts/withdraw.js --withdraw --transferFundsToMaster --masterSafe=$MASTER_SAFE --withdrawalFile="./examples/exampleDepositList.json" --network=$NETWORK_NAME
+npx truffle exec scripts/withdraw.js --masterSafe=$MASTER_SAFE --brackets=0x1,0x2 --tokenIds=1,7 --network=$NETWORK_NAME
 ```
 
-The latter instruction can be split into two independent units, if needed: withdrawing from the exchange to the bracket and transferring funds from the bracket to the master Safe.
+If necessary, for example if some funds are stuck in some of the bracket, the previous command can be split into two independent units: withdrawing from the exchange into the bracket and transferring funds from the bracket into the master Safe.
 
 ```js
-npx truffle exec scripts/withdraw.js --withdraw --masterSafe=$MASTER_SAFE --withdrawalFile="./examples/exampleDepositList.json" --network=$NETWORK_NAME
+npx truffle exec scripts/withdraw.js --withdraw --masterSafe=$MASTER_SAFE --brackets=0x1,0x2 --tokenIds=1,7 --network=$NETWORK_NAME
```

Comment:
Would this flag be more descriptive if it were called `claimOnly`? Just a thought. In fact, I feel like the script itself might benefit from being called `claim_withdraw.js`. 

Furthermore, do we really have a realistic use case for ever wanting the funds to only be put back into the brackets? It almost seems like we shouldn't even bother making this a possibility. Why not bind these operations together so that users never even have to think about whats happening in between.

Reply:
On that very same note, if we are going to keep this "leave funds in limbo" feature, we should probably allow the same for deposits. I am now beginning to think that we should rename the transferApproveDeposit to just "deposit" and try to forget about whats happening in between.